src/Workspaces/CSharp/Portable/LanguageServices/CSharpSyntaxFactsService.cs

```diff
@@ -575,7 +607,7 @@ public bool IsTypeNamedDynamic(SyntaxToken token, SyntaxNode parent)
 
         public bool IsBindableToken(SyntaxToken token)
         {
-            if (this.IsWord(token) || this.IsLiteral(token) || this.IsOperator(token))
+            if (this.IsWord(token) || this.IsLiteral(token) || this.IsOperator(token) || token.Kind() == SyntaxKind.CommaToken)
```

Comment:
The comma is only bindable as part of an `order by` clause (the 1% case). Maybe this check should reflect this:

```CS
(token.Kind() == SyntaxKind.CommaToken && token.Parent.Kind() == SyntaxKind.OrderByClause)
```

Doing so would make sure, that in the 99% case a comma is not categorized as being bindable and would avoid the reporting of false positives for callers of `IsBindableToken`. Other callers of `IsBindableToken` may also show unintended behavior, if every comma is reported as bindable.

Reply:
👍 I like the idea of restricting this for now.

Reply:
agreed.